python/lsst/sims/photUtils/matchUtils.py

```diff
@@ -138,13 +138,9 @@ class matchStar(matchBase):
     the format of those included here.
     """
 
-    def __init__(self, sEDDir = None, kuruczDir = None, mltDir = None, wdDir = None):
+    def __init__(self, kuruczDir = None, mltDir = None, wdDir = None):
 
         """
-        @param [in] sEDDir is a place to specify a different path to a directory that follows the same
-        directory structure as SIMS_SED_LIBRARY. For instance, a different version of the LSST
-        SIMS_SED_LIBRARY.
-
         @param [in] kuruczDir is a place to specify a different path to kurucz SED files than the
         files in the LSST sims_sed_library. If set to None it will default to the LSST library.
         Will probably be most useful for those who want to use loadGalfast without downloading the
```

Comment:
Does this docstring need updating? I don't see how it could default to anything, without getting the SIMS_SED_LIBRARY from the eups package info. 
Was the removal of the SedDir done just to remove the dependency on sims_sed_library?
I would actually think there's a way around this, where we could keep the capability of mapping things automatically (using sims_sed_library + setting the value via eups). If we just do not declare sims_sed_library as a dependency here, but (A) let the user set sedDir, (b) if the user does not set sedDir, then attempt to look for sims_sed_library directory via lsst.utils.getPackageDir, but (c) if that doesn't work either, then only report an error if kurucz/mltDir/wdDir values are None (or let them be set to None, and then complain when the seds are loaded).
Essentially, this last step only is what happens in this implementation, but I'm a little worried that some of the catalogs code might be implicitly using the sedDir + specmap capabilities, and if those are available, it'd be nice to let the user continue to use them (but just not require them, if they're not available).
This does mean that the user has to remember to set up sims_sed_library, if they want to use the backup/non-specification behavior, but I think that's okay given the tradeoff.